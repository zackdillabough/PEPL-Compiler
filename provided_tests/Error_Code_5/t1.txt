POLY A(M, N, O) = M^7 - 6 N M O^4 + 9 M^8 N^3 O^2 - 5 N M^5 O + 23 M^2 N O;
POLY B = x^4 + x^3 + x^2 + x^1 + x^0;
POLY C(P, Q) = P - P + Q - Q;
START
INPUT X;
INPUT Y;
INPUT Z;
C(A, B);
A(X, F, E);
B(1);
B(B(1));
98 34 92 12 3 4
