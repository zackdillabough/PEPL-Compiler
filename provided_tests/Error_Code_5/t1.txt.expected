Error Code 5: 8 8 9 9 